City-of-Helsinki/paatokset-search
# PP-431: Frontpage search

**To test**
- Checkout branch, run `npm i` to install dependencies
- Make sure your `.env` file has this line `REACT_APP_ELASTIC_URL=https://paatokset-proxy-test.agw.arodevtest.hel.fi/` to use the test site's index to make testing easier
- Make sure `public/index.html` has the "frontpage" app selected
- Read code changes to see if anything should be fixed or refactored
- Run `npm start` to open the application and make some searches. The autocomplete should work the same way it does on the decisions search
- When you complete the search, the page should redirect you to `/fi/asia?="[search query]"`
- The this on the swedish and english language versions too with `sv` and `/en` prefixes. They should redirect to:
  - `/sv/arende?s="[query]"`
  - `/en/case?s="[query]"`
